### ISSUE TITLE
nginx error log entries

### ISSUE THREAD

**Issue by meyergru** [Ansuel/tch-nginx-gui#70]
I have a bunch of errors in /var/log/nginx/error.log:

```
2018/07/08 11:09:01 [error] 26265#0: *138 attempt to set ngx.header.HEADER after sending out response headers, client: 192.168.1.97, server: localhost, request: "POST /modals/internet-modal.lp?auto_update=true HTTP/1.1", host: "modemtim", referrer: "https://192.168.1.1/"
```

As there is no direct attempt to set HTTP headers, it must be one of the helpers trying to.

**Comment by Ansuel** [comment]
can you tell me some way to reproduce this?

**Comment by Ansuel** [comment]
Fixed in 818b8aa717c4c9ea4463f442e321db73fef091c7
